cmake/EmscriptenTargets.cmake

```diff
@@ -69,3 +69,14 @@ target_link_libraries(swipl-bundle libswipl)
 add_dependencies(swipl-bundle wasm_preload)
 set_property(TARGET swipl-bundle PROPERTY LINK_DEPENDS
 	     ${POSTJS} ${PREJS})
+
+set(WASM_NO_DATA_BUNDLE_LINK_FLAGS
+    -s SINGLE_FILE)
+join_list(WASM_NO_DATA_BUNDLE_LINK_FLAGS_STRING " " ${WASM_NO_DATA_BUNDLE_LINK_FLAGS} " " ${WASM_SHARED_LINK_FLAGS})
+add_executable(swipl-bundle-no-data ${SWIPL_SRC})
+set_target_properties(swipl-bundle-no-data PROPERTIES
+		      LINK_FLAGS "${WASM_NO_DATA_BUNDLE_LINK_FLAGS_STRING}")
+target_link_libraries(swipl-bundle-no-data libswipl)
+add_dependencies(swipl-bundle-no-data wasm_preload)
```

Comment:
If we have no data, why do we need the preload?

Reply:
Indeed it does nothing - have now removed that line.